worker/include/Channel/UnixStreamSocket.hpp

```diff
@@ -72,6 +72,9 @@ namespace Channel
 		void OnConsumerSocketMessage(ConsumerSocket* consumerSocket, json& jsonMessage) override;
 		void OnConsumerSocketClosed(ConsumerSocket* consumerSocket) override;
 
+	private:
+		void SendImpl(const void* nsPayload, size_t nsPayloadLen);
```

Comment:
Please move this above "Pure virtual methods inherited from ConsumerSocket::Listener" section. That's how we do in all C++ classes.